index.html

```diff
@@ -62,6 +62,177 @@ <h1 class="header-title">I’m Abdallah Glad to see you!</h1>
            </li>    
         </ul>
     </section>
+
+    <section class="works">
+      <div class="card">
+        <img src="images/snapshort_project1.png" alt="portfolio project" width="295"
+         height="220">
+         <h1 class="header-title">Tonic</h1>
+         <ul class="tonic-items">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+         </ul>
+         <p class="sub-title">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <ul class="social-links languages">
+            <li class="languages-li">html</li>
+            <li>css</li>
+            <li>javascript</li>
+         </ul>
+         <button  class="p-btn">See Project</button>
```

Comment:
- [ ] Kindly update this button and all the other buttons to make them match the ones on Figma, this includes `background-color` and `font-size`, and button interactions (hover, pressed...)

## Your design:
![image](https://user-images.githubusercontent.com/50721479/192098531-86885e57-61fd-4eac-8a85-f37a76b1f4fe.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192098545-beb09145-0c00-4217-82a7-7d397f88df4d.png)

![image](https://user-images.githubusercontent.com/50721479/192098536-84dd6435-220e-445c-805a-25fc166bc908.png)